azure_arc_servers_jumpstart/docs/arc_policies_mma.md

```diff
@@ -0,0 +1,113 @@
+# Overview
+
+The following README will guide you on how to use Arc for servers to assign Azure Policies to VMs outside of Azure, whether they are on-premises or other clouds. With this feature you can now use Azure Policies to audit settings in the operating system of an Azure Arc connected servers, if a setting is not compliant you can also trigger a remediation task. 
+
```

Comment:
```suggestion
The following README will guide you on to assign Azure Policies to servers deployed outside of Azure (whether on-premises or in other clouds) while leveraging Azure Arc. With this feature, you can now use Azure Policies to audit settings in the operating system of an Azure Arc connected server, and in case found as non-compliant, you can also trigger a remediation task. 

```